content/advocacy/mvmha/index.md

```diff
@@ -29,22 +29,32 @@ The MVMHA includes all six mobile home communities in Mountain View:
 - Moorpark Mobile Home Park
 - Moffett Mobile Home Park
 
-## How We Got Mobile Home Rents Stabilized
+## How We Got Mobile Home Space Rents Stabilized
```

Comment:
The non-space rents are also stabilized.

Reply:
Removed "Space"